app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,64 @@
+prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
+  # Vertical Layout
+  user = @physical_ticket.user
+  ticket_purchase = @physical_ticket.ticket_purchase
```

Comment:
this is only used one, just wrote `@physical_ticket.ticket_purchase` where it is used and remove the variable!